<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubtitleConfig</class>
 <widget class="QDialog" name="SubtitleConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Subtitle Configuration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="srt_editor_resources.qrc">
    <normaloff>:/mainwindow/assets/icon.png</normaloff>:/mainwindow/assets/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Speech</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Model</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="speechModels">
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Voice</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="speechVoices">
                <property name="layoutDirection">
                 <enum>Qt::RightToLeft</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Speed</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="speechSpeed">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Instructions</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="openaiSpeechInstructions"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="0,1">
              <item>
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Language</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="speechLanguage">
                <property name="layoutDirection">
                 <enum>Qt::RightToLeft</enum>
                </property>
                <item>
                 <property name="text">
                  <string>Vietnamese</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>English</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Japanese</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Chinese</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Korean</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Format</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="speechFileFormat">
                <property name="layoutDirection">
                 <enum>Qt::RightToLeft</enum>
                </property>
                <item>
                 <property name="text">
                  <string>MP3</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>AAC</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>WAV</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Translate</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,1">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Model</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="translateModels">
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="0,1">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Source Language</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="srcLanguage">
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <item>
               <property name="text">
                <string>Vietnamese</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>English</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Japanese</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Chinese</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Korean</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="0,1">
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Destination Language</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="dstLanguage">
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <item>
               <property name="text">
                <string>English</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Vietnamese</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Japanese</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Chinese</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Korean</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="srt_editor_resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SubtitleConfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SubtitleConfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
